templates/4a_compensate_image_data.ipynb

```diff
@@ -5,9 +5,25 @@
    "id": "ca8b60ef",
```

Comment:
<p>Crop these images to just the upper left hand 3x3 or 4x4 tiles, so its easier to see the data. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>